microsoft/testsuites/core/floppy.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from assertpy import assert_that
+
+from lisa import RemoteNode, TestCaseMetadata, TestSuite, TestSuiteMetadata
+
+
+@TestSuiteMetadata(
+    area="core",
+    category="functional",
+    description="""
+    This test suite ensures the floppy module is disabled.
+    """,
+)
+class Floppy(TestSuite):
```

Comment:
we can make a modules test suites, besides check floppy module, we have other modules related case can be added.